samples/sampler/stories/qa/VirtualList.js

```diff
@@ -542,3 +542,60 @@ WithContainerItemsHaveSpottableControls.storyName = 'with container items have s
 WithContainerItemsHaveSpottableControls.parameters = {
 	propTables: [Config]
 };
+
+const fixedItemSizes = new Array(16).fill(ri.scale(390));
+const variableItemSizes = fixedItemSizes.map((size, index) => {
+	return index % 2  ? size * 2  : size;
+});
+
+class VirtualListWithChangingSizes extends Component {
```

Comment:
Is there a specific reason why you chose class over functional component?
If it's a new one, let's make it functional component.

Reply:
I updated.